### PR TITLE
Support for authorization configuration

### PR DESCRIPTION
**Pull request by martin1cerny**
Added configuration options where developers can define client ID and redirect URL. Added static redirection handler static file. It can be served in the same way as UI and JSON schema. Added test.

### CODE REVIEW

**Comment by swasun99** [comment]
@martin1cerny - Hi Martin, in which version will we have this feature available?

**Comment by martin1cerny** [comment]
That depends on @vitalybibikov.

File: src/AzureFunctions.Extensions.Swashbuckle/AzureFunctions.Extensions.Swashbuckle/Settings/SwaggerDocOptions.cs [hunk 1]
```diff
@@ -23,5 +23,10 @@ public class SwaggerDocOptions
         public OpenApiSpecVersion SpecVersion { get; set; } = OpenApiSpecVersion.OpenApi3_0;
 
         public Action<SwaggerGenOptions> ConfigureSwaggerGen { get; set; }
+
+        public string ClientId { get; set; } = "";
```
Comment (vitalybibikov) [inline src/AzureFunctions.Extensions.Swashbuckle/AzureFunctions.Extensions.Swashbuckle/Settings/SwaggerDocOptions.cs:27]: I guess it's better to use string?  here

**Review by vitalybibikov — COMMENTED** [review]
Nice PR!

Please, can you merge the PR into dev branch, instead of master?

I'm going to release all the -beta versions out of dev one, when tested by "beta users", they are going to be merged into master and be available as a normal package.
Thanks!

**Comment by martin1cerny** [comment]
Sure, I'll create a PR to dev tomorrow. I created PR to master because it was ahead of dev.

File: src/AzureFunctions.Extensions.Swashbuckle/AzureFunctions.Extensions.Swashbuckle/Settings/SwaggerDocOptions.cs (hunk 1 above)
Comment (vitalybibikov) [reply src/AzureFunctions.Extensions.Swashbuckle/AzureFunctions.Extensions.Swashbuckle/Settings/SwaggerDocOptions.cs:27]: No, I meant "string?" =) 

public string? OAuthClientId { get; set; }

public string? OAuth2RedirectPath { get; set; }

**Comment by martin1cerny** [comment]
Closing this PR, new PR #64 created for `dev` branch.

**Pull request closed without merging**